modules/op-sample-example-pipeline-tutorial.adoc

```diff
@@ -0,0 +1,49 @@
+[id='op-sample-example-pipeline-tutorial_{context}']
+= Sample example from pipeline tutorial
+
+We are using https://github.com/openshift/pipelines-tutorial[pipelines tutorial] as a sample example.
```

Comment:
link:https://github.com/openshift/pipelines-tutorial[pipelines tutorial]

Same for other external links.